3 tests are failing here:

(1) `TestAlreadyRunningJobsAreHandledProperly`. in `pkg/upgrade/upgrademanager/upgrademanager_test`
```
=== RUN   TestAlreadyRunningJobsAreHandledProperly
    manager_external_test.go:120:
          Error Trace:  pkg/upgrade/upgrademanager/upgrademanager_test_test/pkg/upgrade/upgrademanager/manager_external_test.go:120
          Error:        Received unexpected error:
                        sql: no rows in result set
```
(2) `TestPauseMigration`. in `pkg/upgrade/upgrademanager/upgrademanager_test`
```
=== RUN   TestPauseMigration
    manager_external_test.go:464: error scanning '&{<nil> 0xc003f80200}': sql: no rows in result set
```
(3) `TestServerStartupGuardrails`. in `github.com/cockroachdb/cockroach/pkg/ccl/serverccl`
```
I230405 05:30:21.943950 189123 upgrade/upgrademanager/manager.go:168  [T1,n1] 1170  running permanent upgrades up to version: 1000022.2
I230405 05:30:21.944731 189876 sql/temporary_schema.go:505  [T1,n1] 1171  running temporary object cleanup background job
I230405 05:30:21.945053 189877 sql/sqlstats/persistedsqlstats/provider.go:129  [T1,n1] 1172  starting sql-stats-worker with initial delay: 10m15.992188656s
I230405 05:30:21.946647 189895 kv/kvclient/rangefeed/rangefeedcache/watcher.go:335  [T1,n1] 1173  system-config-cache: established range feed cache
I230405 05:30:21.947169 189864 kv/kvserver/replica_rangefeed.go:701  [T1,n1,rangefeed=table-stats-cache,mux_n=1,gen=2,r21/1:/Table/2{0-1},s1] 1174  RangeFeed closed timestamp is empty
I230405 05:30:21.950611 189876 sql/temporary_schema.go:555  [T1,n1] 1175  found 0 temporary schemas
I230405 05:30:21.950720 189876 sql/temporary_schema.go:558  [T1,n1] 1176  early exiting temporary schema cleaner as no temporary schemas were found
I230405 05:30:21.950747 189876 sql/temporary_schema.go:559  [T1,n1] 1177  completed temporary object cleanup job
I230405 05:30:21.950765 189876 sql/temporary_schema.go:640  [T1,n1] 1178  temporary object cleaner next scheduled to run at 2023-04-05 06:00:21.944702855 +0000 UTC m=+1944.305053323
E230405 05:30:21.994611 189894 spanconfig/spanconfigmanager/manager.go:141  [T1,n1] 1179  error starting auto span config reconciliation job: get-jobs: system-jobs-scan: relation "system.job_info" does not exist
I230405 05:30:22.008682 189123 upgrade/upgrademanager/manager.go:229  [T1,n1] 1180  the last permanent upgrade (v22.2-98) does not appear to have completed; attempting to run all upgrades
I230405 05:30:22.009905 189123 upgrade/upgrademanager/manager.go:270  [T1,n1] 1181  running permanent upgrade for version 0.0-2
    test_server_shim.go:331: migration-manager-find-jobs: system-jobs-scan: relation "system.job_info" does not exist
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1

```

--- a/docs/generated/settings/settings-for-tenants.txt
+++ b/docs/generated/settings/settings-for-tenants.txt
@@ -293,4 +293,4 @@ trace.opentelemetry.collector	string		address of an OpenTelemetry trace collecto
 trace.snapshot.rate	duration	0s	if non-zero, interval at which background trace snapshots are captured
 trace.span_registry.enabled	boolean	true	if set, ongoing traces can be seen at https://<ui>/#/debug/tracez
 trace.zipkin.collector	string		the address of a Zipkin instance to receive traces, as <host>:<port>. If no port is specified, 9411 will be used.
-version	version	1000022.2-98	set the active cluster version in the format '<major>.<minor>'
+version	version	22.2-98	set the active cluster version in the format '<major>.<minor>'

--- a/docs/generated/settings/settings.html
+++ b/docs/generated/settings/settings.html
@@ -245,6 +245,6 @@
 <tr><td><div id="setting-trace-snapshot-rate" class="anchored"><code>trace.snapshot.rate</code></div></td><td>duration</td><td><code>0s</code></td><td>if non-zero, interval at which background trace snapshots are captured</td></tr>
 <tr><td><div id="setting-trace-span-registry-enabled" class="anchored"><code>trace.span_registry.enabled</code></div></td><td>boolean</td><td><code>true</code></td><td>if set, ongoing traces can be seen at https://&lt;ui&gt;/#/debug/tracez</td></tr>
 <tr><td><div id="setting-trace-zipkin-collector" class="anchored"><code>trace.zipkin.collector</code></div></td><td>string</td><td><code></code></td><td>the address of a Zipkin instance to receive traces, as &lt;host&gt;:&lt;port&gt;. If no port is specified, 9411 will be used.</td></tr>
-<tr><td><div id="setting-version" class="anchored"><code>version</code></div></td><td>version</td><td><code>1000022.2-98</code></td><td>set the active cluster version in the format &#39;&lt;major&gt;.&lt;minor&gt;&#39;</td></tr>
+<tr><td><div id="setting-version" class="anchored"><code>version</code></div></td><td>version</td><td><code>22.2-98</code></td><td>set the active cluster version in the format &#39;&lt;major&gt;.&lt;minor&gt;&#39;</td></tr>
 </tbody>
 </table>

--- a/pkg/cli/testdata/declarative-rules/deprules
+++ b/pkg/cli/testdata/declarative-rules/deprules
@@ -1,6 +1,6 @@
 dep
 ----
-debug declarative-print-rules 1000022.2-98 dep
+debug declarative-print-rules 22.2-98 dep
 deprules
 ----
 - name: 'CheckConstraint transitions to ABSENT uphold 2-version invariant: PUBLIC->VALIDATED'

--- a/pkg/cli/testdata/declarative-rules/invalid_version
+++ b/pkg/cli/testdata/declarative-rules/invalid_version
@@ -4,4 +4,4 @@ invalid_version
 debug declarative-print-rules 1.1 op
 unsupported version number, the supported versions are: 
  latest
- 1000022.2
+ 22.2

--- a/pkg/cli/testdata/declarative-rules/oprules
+++ b/pkg/cli/testdata/declarative-rules/oprules
@@ -1,6 +1,6 @@
 op
 ----
-debug declarative-print-rules 1000022.2-98 op
+debug declarative-print-rules 22.2-98 op
 rules
 ----
 []

--- a/pkg/clusterversion/cockroach_versions.go
+++ b/pkg/clusterversion/cockroach_versions.go
@@ -914,10 +914,9 @@ var rawVersionsSingleton = keyedVersions{
 // frozen. It can be forced to a specific value in two circumstances:
 // 1. forced to `false` on development branches: this is used for
 // upgrade testing purposes and should never be done in real clusters;
-// 2. forced to `false` on release branches: this allows running a
+// 2. forced to `true` on release branches: this allows running a
 // release binary in a dev cluster.
-var developmentBranch = !envutil.EnvOrDefaultBool("COCKROACH_TESTING_FORCE_RELEASE_BRANCH", false) ||
-	envutil.EnvOrDefaultBool("COCKROACH_FORCE_DEV_VERSION", false)
+var developmentBranch = envutil.EnvOrDefaultBool("COCKROACH_FORCE_DEV_VERSION", false)
 
 const (
 	// finalVersion should be set on a release branch to the minted final cluster

--- a/pkg/sql/logictest/testdata/logic_test/crdb_internal
+++ b/pkg/sql/logictest/testdata/logic_test/crdb_internal
@@ -566,7 +566,7 @@ select crdb_internal.get_vmodule()
 query T
 select regexp_replace(crdb_internal.node_executable_version()::string, '(-\d+)?$', '');
 ----
-1000022.2
+22.2
 
 query ITTT colnames
 select node_id, component, field, regexp_replace(regexp_replace(value, '^\d+$', '<port>'), e':\\d+', ':<port>') as value from crdb_internal.node_runtime_info
@@ -720,7 +720,7 @@ SELECT * FROM crdb_internal.check_consistency(true, b'\x02', b'\x04')
 query T
 select regexp_replace(crdb_internal.node_executable_version()::string, '(-\d+)?$', '');
 ----
-1000022.2
+22.2
 
 user root
 

--- a/pkg/sql/logictest/testdata/logic_test/mixed_version_can_login
+++ b/pkg/sql/logictest/testdata/logic_test/mixed_version_can_login
@@ -21,7 +21,7 @@ SELECT crdb_internal.create_tenant(1001)
 upgrade 1
 
 query B
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 
@@ -34,7 +34,7 @@ SELECT crdb_internal.node_executable_version()
 user testuser nodeidx=0
 
 query B
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 
@@ -42,7 +42,7 @@ true
 user root nodeidx=1
 
 query B
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 
@@ -65,7 +65,7 @@ SELECT crdb_internal.node_executable_version()
 upgrade 2
 
 query B
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 

--- a/pkg/sql/logictest/testdata/logic_test/mixed_version_range_tombstones
+++ b/pkg/sql/logictest/testdata/logic_test/mixed_version_range_tombstones
@@ -22,7 +22,7 @@ false
 upgrade 0
 
 query B nodeidx=0
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 

--- a/pkg/sql/logictest/testdata/logic_test/mixed_version_role_members_user_ids
+++ b/pkg/sql/logictest/testdata/logic_test/mixed_version_role_members_user_ids
@@ -47,7 +47,7 @@ upgrade 1
 # Test that there are no problems creating role memberships on a mixed 22.2/23.1 cluster.
 
 query B nodeidx=1
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 
@@ -83,16 +83,16 @@ upgrade 2
 # Verify that all nodes are now running 23.1 binaries.
 
 query B nodeidx=0
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 
 query B nodeidx=1
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true
 
 query B nodeidx=2
-SELECT crdb_internal.node_executable_version() SIMILAR TO '1000022.2-%'
+SELECT crdb_internal.node_executable_version() SIMILAR TO '22.2-%'
 ----
 true

